Documentation/RC1SupportedScenarios.md

```diff
@@ -0,0 +1,24 @@
+WCF in RC1
+===========================
+
+WCF Features
+------------
+
+
+![Image](http://findicons.com/files/icons/573/must_have/16/check.png) -- Supported: Release quality
+
+![Image](http://findicons.com/files/icons/1155/diagona/16/104.png) --  Supported: Not fully tested
+
+![Image](http://findicons.com/files/icons/573/must_have/16/add.png) -- Not Yet Supported: Future release
+
+![Image](http://findicons.com/files/icons/573/must_have/16/delete.png) -- Not Supported: Will not support
+
```

Comment:
I think it would be a good convention that any icon other than "supported" should have a hyperlink the user can follow for more information.  It might be a link to an issue or a markdown file.  I think we should also adopt a simple naming convention for the hyperlink's display name.  Something simple like "?" or "why?".  As a user, I think the first thing I would want to know about a "not supported" item is "why not?" and "when?"

We can get the basic table checked in and fill in the hyperlinks afterwards (though maybe one hyperlink on the initial checkin would help show the pattern to follow.